src/modules/mavlink/mavlink_main.cpp

```diff
@@ -2475,6 +2481,26 @@ Mavlink::stream_command(int argc, char *argv[])
 	return OK;
 }
 
+void
+Mavlink::set_boot_complete()
+{
+	_boot_complete = true;
+
+#ifdef __PX4_POSIX
+	Mavlink *inst;
+	LL_FOREACH(::_mavlink_instances, inst) {
+		if ((inst->get_mode() != MAVLINK_MODE_ONBOARD) &&
+			(!inst->broadcast_enabled()) &&
+			((inst->get_protocol() == UDP) || (inst->get_protocol() == TCP))) {
+			printf("\n\n   **********   NETWORK BROADCAST NOT ENABLED   **********\n" \
```

Comment:
It was a conscious decision to not use PX4 info because I don't want the process output clutter the message. I also want this to be very visible because users will else not see it (and we could just leave it away in the first place).